pkg/acceptance/bettertestspoc/config/hcl_config_provider.go

```diff
@@ -79,6 +79,20 @@ func fixBlockArguments(s string) (string, error) {
 	return argumentRegex.ReplaceAllString(s, `$1$2$3$4`), nil
 }
 
+// TODO [SNOW-1501905]: fix new lines replacement totally in this method, consider better placeholders (now there can't be multilinekeys for private key)
+func fixMultilinePrivateKey(s string) (string, error) {
```

Comment:
So it's not clear whether we should use this function at all.

Reply:
I will change the comment to a more readable one in the next PRs